.github/workflows/testadapters.yaml

```diff
@@ -108,14 +108,14 @@ jobs:
       - name: Check Meshery pod logs
         run: |
           SECONDS=0
-          end=$((SECONDS+150))
+          end=$((SECONDS+300))
```

Comment:
Ugh. More sleeping.

@hexxdump have you arrived at a solution for `mesheryctl system start`'s sleeping? // @piyushsingariya

Reply:
I am not sleeping here. We are polling. This time is just the timeout, untill when to POLL

Reply:
Ok. So, this is sleeping with a chance for early exit. ;)

How can we do this better? @hexxdump?

Reply:
No, not sleeping, we keep running a specific command and keep checking if the output is as desired. When we get the desired output, we exit. This time is just the max timeout after which we dont further run that specific command

Reply:
Related - https://github.com/meshery/meshery/issues/4726

Reply:
My sarcasm wasn't thick enough. I'm saying that this polling approach is marginally better than sleeping and in its basic essence, the same thing.